README.md

```diff
@@ -2,26 +2,89 @@
 
 [![Version](https://img.shields.io/maven-central/v/io.quarkiverse.argocd/quarkus-argocd?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.argocd/quarkus-argocd-parent)
 
-## Welcome to Quarkiverse!
+## Features
 
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
+- Generate the ArgoCD Application Kubernetes CR for the Quarkus application
+- Command Line interface to install / unistall and list Application(s)
+- Integration with Quarkus Helm
 
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
 
-You can find the basic info, Quarkiverse policies and conventions in [the Quarkiverse wiki](https://github.com/quarkiverse/quarkiverse/wiki).
+## Building
 
-In case you are creating a Quarkus extension project for the first time, please follow [Building My First Extension](https://quarkus.io/guides/building-my-first-extension) guide.
+To build the extension use the following command:
```

Comment:
Maybe use `To build this extension which is also a plugin, use ....`

Reply:
We've already mentioned that the project provides a plugin, above